docs/notebooks/intro.py

```diff
@@ -26,7 +26,6 @@
 import pandas as pd
 import tensorflow as tf
 
-tf.keras.backend.set_floatx("float64")
```

Comment:
@vdutor I have tested only this notebook that all works fine, are notebooks executed in some tests?

Reply:
Yes they are.

Reply:
then we should be fine, all tests are passing